reference/concepts/assignment.md

```diff
@@ -1,5 +1,28 @@
 # Assignment
 
+## The Concept
+
 An assignment statement sets and/or re-sets the value stored in the storage location(s) denoted by a variable name; in other words, it copies a value into the [variable][concept-variables]. In most imperative programming languages, the assignment statement (or expression) is a fundamental construct.
 
+## What to cover
+
+- **How to assign values** How to assign a value to a variable.
+- **How to update a value** How to update a variable's value.
```

Comment:
Looking at the exercise can we say that updating the variable's value is covered since only the test does it?

Reply:
Yeah, this is a hairy point, which is indeed problematic. The updated exercise will exclude them.